### PR TITLE
Tdykes.response.mapmarkers.realtionship

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
destination on event response now points to the event location directly.

Updated UI to work, but in next rework of the events screen it will take advantage of this, currently it just doesnt do anything special with it, I just made it not break

land #201 first

### CODE REVIEW

**Comment by OSPFNeighbour** [comment]
This is GTG if I could get some stamps. @sdunster 

```
{
                    "user": {
                      "username": "kiama1"
                    },
                    "destination": {
                      "id": 1,
                      "name": "lhq",
                      "detail": "Local Head Quarters"
                    }
                  }
```

File: server/src/logic.js [hunk 1]
```diff
@@ -328,6 +328,7 @@ export const getHandlers = ({ models, creators: Creators }) => {
         const {
           id,
         } = args.response;
+        console.log(args);
```
Comment (sjmcculloch) [inline server/src/logic.js]: remove consol.log?

**Comment by sjmcculloch** [comment]
It looks OK to me - I'll leave to @sdunster  to stamp for approval.

File: server/src/logic.js (hunk 1 above)
Comment (OSPFNeighbour) [reply server/src/logic.js]: thx. removed.

**Comment by OSPFNeighbour** [comment]
land #201 first

File: server/src/logic.js [hunk 2]
```diff
@@ -353,6 +361,9 @@ export const getHandlers = ({ models, creators: Creators }) => {
       user(response) {
         return response.getUser();
       },
+      eventLocations(response) {
```
Comment (sdunster) [inline server/src/logic.js]: i think you should name this `destination` to match the schema. at the very least it should not be plural

File: server/src/resolvers.js [hunk 3]
```diff
@@ -79,6 +79,9 @@ export const getResolvers = (handlers) => {
       user(response, args, ctx) {
         return eventResponseHandler.user(response, args, ctx);
       },
+      destination(response, args, ctx) {
+        return eventResponseHandler.eventLocations(response, args, ctx);
```
Comment (sdunster) [inline server/src/resolvers.js]: change this to `destination` as well

**Pull request merged**